### PR TITLE
Split the brain board update docs into a separate page.

### PR DESCRIPTION
**Pull request by trickeydan**
We will be adding extra content to both the brain board and this new page soon, so let's split them up before the pages get too long.

### CODE REVIEW

File: kit/brain_board/updating-the-brain.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (RealOrangeOne) [inline kit/brain_board/updating-the-brain.md:1]: Suggestion: The updates list page is pretty small - perhaps these update instructions should go there instead? That way there's a single page for "what are the updates" and "how do I install them".

Comment (trickeydan) [reply kit/brain_board/updating-the-brain.md:1]: This page will get much longer when we add info about the file based update system.

File: kit/brain_board/updating-the-brain.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+layout: page
+title: Updating The Brain
+---
+
+{% assign latest_version = site.data.kit_versions | first %}
+{% assign latest_version = latest_version.version | default: "2022.1.0" %}
+
+# Updating The Brain
+
+### Flashing SD card
```
Comment (sedders123) [inline kit/brain_board/updating-the-brain.md:11]: Does the heading structure still make sense now its on its own page?

I would have thought Flashing SD Card would be h2, Etcher h3 and Flashing h4.

**Comment by trickeydan** [comment]
We're dropping rauc this year.

**Pull request closed without merging**